test/libsolidity/syntaxTests/literalSuffixes/parameters/exponent_in_range.sol

```diff
@@ -31,3 +31,12 @@ contract C {
     }
 }
+// ----
+// TypeError 3123: (473-482): The exponent parameter of a literal suffix function must be unsigned. Exponent is always either zero or a negative power of 10 but the parameter represents its absolute value.
+// TypeError 3123: (511-521): The exponent parameter of a literal suffix function must be unsigned. Exponent is always either zero or a negative power of 10 but the parameter represents its absolute value.
+// TypeError 3123: (551-562): The exponent parameter of a literal suffix function must be unsigned. Exponent is always either zero or a negative power of 10 but the parameter represents its absolute value.
+// TypeError 3123: (594-677): The exponent parameter of a literal suffix function must be unsigned. Exponent is always either zero or a negative power of 10 but the parameter represents its absolute value.
+// TypeError 3123: (812-896): The exponent parameter of a literal suffix function must be unsigned. Exponent is always either zero or a negative power of 10 but the parameter represents its absolute value.
+// TypeError 3123: (1030-1115): The exponent parameter of a literal suffix function must be unsigned. Exponent is always either zero or a negative power of 10 but the parameter represents its absolute value.
+// TypeError 3123: (1288-1297): The exponent parameter of a literal suffix function must be unsigned. Exponent is always either zero or a negative power of 10 but the parameter represents its absolute value.
+// TypeError 3123: (1347-34120): The exponent parameter of a literal suffix function must be unsigned. Exponent is always either zero or a negative power of 10 but the parameter represents its absolute value.
```

Comment:
This test is supposed to test the range of the exponent. It should not be failing. Since signed exponent variants are no longer valid, I think it's best to just remove them. Now the `signed_exponent.sol` covers them.